### ISSUE TITLE
Remove render-blocking JavaScript:

### ISSUE THREAD

**Issue by GaryMilburn** [GoogleChrome/lighthouse#7265]
• Remove render-blocking JavaScript:
https://www.milburnplumbingservices.com.au/gdpr/gdprscript.js?buildTime=1550256872&hasRemindMe=true&stealth=false
https://cdn2.editmysite.com/js/lang/en/stl.js?buildTime=1550256872&
https://cdn2.editmysite.com/js/site/main.js?buildTime=1550256872
https://cdn2.editmysite.com/js/lang/en/stl.js?buildTime=1550256872&
• Optimize CSS Delivery of the following:
https://cdn2.editmysite.com/css/sites.css?buildTime=1550256872
https://cdn2.editmysite.com/css/old/fancybox.css?1550256872
https://cdn2.editmysite.com/css/social-icons.css?buildtime=1550256872
https://www.milburnplumbingservices.com.au/files/main_style.css?1550472695
https://fonts.googleapis.com/css?family=Montserrat:400,700&subset=latin,latin-ext

**Comment by patrickhulce** [comment]
Did you have a question about these results @GaryMilburn?

**Comment by GaryMilburn** [comment]
Yes patrickhuice l need to know how to remove the render blocking javaSripts from my Website and the need to Optimize CSS delivery of the the following scripts.

**Comment by patrickhulce** [comment]
There are "Learn More" links in the description of each opportunity that provide more information on how to do follow their advice.

![image](https://user-images.githubusercontent.com/2301202/52977080-89ff5f80-3391-11e9-8ef6-ed4cdf371b5f.png)

**Comment by GaryMilburn** [comment]
> Did you have a question about these results @GaryMilburn?

I want them fixed

**Comment by GaryMilburn** [comment]
fix these issues

**Comment by GaryMilburn** [comment]
<!DOCTYPE html>

<html lang="en"><head><script type="text/javascript" async="" src="https://www.google-analytics.com/analytics.js"></script><script type="text/javascript" async="" src="https://www.google-analytics.com/gtm/js?id=GTM-WX8BKJW&amp;t=gtag_UA_88378966_1&amp;cid=526033465.1551327791"></script><script async="" src="//cdn2.editmysite.com/js/wsnbn/snowday262.js"></script><script type="text/javascript" async="" src="https://ssl.google-analytics.com/ga.js"></script><script async="" src="https://www.google-analytics.com/analytics.js"></script><script async="" src="//cdn2.editmysite.com/js/wsnbn/snowday261.js"></script><script src="/gdpr/gdprscript.js?buildTime=1551314940&amp;hasRemindMe=true&amp;stealth=false"></script>

		<title>Milburn Plumbing Services - Plumbing and Gasfitting Services</title><meta property="og:site_name" content="Milburn Plumbing Services" />

<meta property="og:title" content="Plumbing and Gasfitting Services" />

<meta property="og:description" content="Milburn Plumbing and Gas fitting services has you covered regarding your Plumbing and Gas fitting needs and concerns." />

<meta property="og:url" content="https://www.milburnplumbingservices.com.au/" />



<meta name="description" content="Milburn Plumbing and Gas fitting services has you covered regarding your Plumbing and Gas fitting needs and concerns." />

<meta name="keywords" content="sewer installation morwell, plumber morwell, plumbing maintenance morwell, plumbing and gasfitting services morwell, sanitary drainage plumber morwell, install plumbing fixtures morwell,plumbing installation and repair service morwell,plumbers morwell" />



	<meta name="description" content=" Milburn Plumbing Services is based in Morwell and can do plumbing and gas fitting for Gippsland and the Latrobe Valley" />

	<meta name="google-site-verification" content="Df43dk7cP6PfUyNR4hgC4HS8J--FE6sryXt0fIR1DAc" />

	<meta name="msvalidate.01" content="44B56F626172388C0E511D0472CE3C5C" />

	<meta name="viewport" content="width=device-width, initial-scale=1.0" />



		

<title>Milburn Plumbing Services</title>

<meta name="description" content=" Milburn Plumbing Services is based in Morwell and can do plumbing and gas fitting for Gippsland and the Latrobe Valley" />





<meta itemprop="name" content="Milburn Plumbing Services" />

<meta itemprop="description" content=" Milburn Plumbing Services is based in Morwell and can do plumbing and gas fitting for Gippsland and the Latrobe Valley" />

<meta itemprop="image" content="http://www.milburnplumbingservices.com.au/uploads/9/5/0/2/95028178/plumbing-licence-min_4_orig.jpg" />





<meta property="og:url" content="https://www.milburnplumbingservices.com.au" />

<meta property="og:type" content="website" />

<meta property="og:title" content="Milburn Plumbing Services" />

<meta property="og:description" content=" Milburn Plumbing Services is based in Morwell and can do plumbing and gas fitting for Gippsland and the Latrobe Valley" />

<meta property="og:image" content="http://www.milburnplumbingservices.com.au/uploads/9/5/0/2/95028178/plumbing-licence-min_4_orig.jpg" />





<meta name="twitter:card" content="summary_large_image" />

<meta name="twitter:title" content="Milburn Plumbing Services" />

<meta name="twitter:description" content=" Milburn Plumbing Services is based in Morwell and can do plumbing and gas fitting for Gippsland and the Latrobe Valley" />

<meta name="twitter:image" content="http://www.milburnplumbingservices.com.au/uploads/9/5/0/2/95028178/plumbing-licence-min_4_orig.jpg" />

<meta name="google-site-verification" content="Df43dk7cP6PfUyNR4hgC4HS8J--FE6sryXt0fIR1DAc" />

<meta name="msvalidate.01" content="44B56F626172388C0E511D0472CE3C5C" />

<meta name="p:domain_verify" content="26b507d18896c398bc273ba39d2b692d" />

		<link id="wsite-base-style" rel="stylesheet" type="text/css" href="//cdn2.editmysite.com/css/sites.css?buildTime=1551314940" />

<link rel="stylesheet" type="text/css" href="//cdn2.editmysite.com/css/old/fancybox.css?1551314940" />

<link rel="stylesheet" type="text/css" href="//cdn2.editmysite.com/css/social-icons.css?buildtime=1551314940" media="screen,projection" />

<link rel="stylesheet" type="text/css" href="/files/main_style.css?1551321416" title="wsite-theme-css" />

<link href="//fonts.googleapis.com/css?family=Montserrat:400,700&amp;subset=latin,latin-ext" rel="stylesheet" type="text/css" />



<style type="text/css">

.wsite-elements.wsite-not-footer:not(.wsite-header-elements) div.paragraph, .wsite-elements.wsite-not-footer:not(.wsite-header-elements) p, .wsite-elements.wsite-not-footer:not(.wsite-header-elements) .product-block .product-title, .wsite-elements.wsite-not-footer:not(.wsite-header-elements) .product-description, .wsite-elements.wsite-not-footer:not(.wsite-header-elements) .wsite-form-field label, .wsite-elements.wsite-not-footer:not(.wsite-header-elements) .wsite-form-field label, #wsite-content div.paragraph, #wsite-content p, #wsite-content .product-block .product-title, #wsite-content .product-description, #wsite-content .wsite-form-field label, #wsite-content .wsite-form-field label, .blog-sidebar div.paragraph, .blog-sidebar p, .blog-sidebar .wsite-form-field label, .blog-sidebar .wsite-form-field label {}

#wsite-content div.paragraph, #wsite-content p, #wsite-content .product-block .product-title, #wsite-content .product-description, #wsite-content .wsite-form-field label, #wsite-content .wsite-form-field label, .blog-sidebar div.paragraph, .blog-sidebar p, .blog-sidebar .wsite-form-field label, .blog-sidebar .wsite-form-field label {}

.wsite-elements.wsite-footer div.paragraph, .wsite-elements.wsite-footer p, .wsite-elements.wsite-footer .product-block .product-title, .wsite-elements.wsite-footer .product-description, .wsite-elements.wsite-footer .wsite-form-field label, .wsite-elements.wsite-footer .wsite-form-field label{}

.wsite-elements.wsite-not-footer:not(.wsite-header-elements) h2, .wsite-elements.wsite-not-footer:not(.wsite-header-elements) .product-long .product-title, .wsite-elements.wsite-not-footer:not(.wsite-header-elements) .product-large .product-title, .wsite-elements.wsite-not-footer:not(.wsite-header-elements) .product-small .product-title, #wsite-content h2, #wsite-content .product-long .product-title, #wsite-content .product-large .product-title, #wsite-content .product-small .product-title, .blog-sidebar h2 {}

#wsite-content h2, #wsite-content .product-long .product-title, #wsite-content .product-large .product-title, #wsite-content .product-small .product-title, .blog-sidebar h2 {}

.wsite-elements.wsite-footer h2, .wsite-elements.wsite-footer .product-long .product-title, .wsite-elements.wsite-footer .product-large .product-title, .wsite-elements.wsite-footer .product-small .product-title{}

#wsite-title {}

.wsite-menu-default a {}

.wsite-menu a {}

.wsite-image div, .wsite-caption {}

.galleryCaptionInnerText {}

.fancybox-title {}

.wslide-caption-text {}

.wsite-phone {}

.wsite-headline,.wsite-header-section .wsite-content-title {}

.wsite-headline-paragraph,.wsite-header-section .paragraph {}

.wsite-button-inner {}

.wsite-not-footer blockquote {}

.wsite-footer blockquote {}

.blog-header h2 a {}

#wsite-content h2.wsite-product-title {}

.wsite-product .wsite-product-price a {}

@media screen and (min-width: 767px) {.wsite-elements.wsite-not-footer:not(.wsite-header-elements) div.paragraph, .wsite-elements.wsite-not-footer:not(.wsite-header-elements) p, .wsite-elements.wsite-not-footer:not(.wsite-header-elements) .product-block .product-title, .wsite-elements.wsite-not-footer:not(.wsite-header-elements) .product-description, .wsite-elements.wsite-not-footer:not(.wsite-header-elements) .wsite-form-field label, .wsite-elements.wsite-not-footer:not(.wsite-header-elements) .wsite-form-field label, #wsite-content div.paragraph, #wsite-content p, #wsite-content .product-block .product-title, #wsite-content .product-description, #wsite-content .wsite-form-field label, #wsite-content .wsite-form-field label, .blog-sidebar div.paragraph, .blog-sidebar p, .blog-sidebar .wsite-form-field label, .blog-sidebar .wsite-form-field label {}

#wsite-content div.paragraph, #wsite-content p, #wsite-content .product-block .product-title, #wsite-content .product-description, #wsite-content .wsite-form-field label, #wsite-content .wsite-form-field label, .blog-sidebar div.paragraph, .blog-sidebar p, .blog-sidebar .wsite-form-field label, .blog-sidebar .wsite-form-field label {}

.wsite-elements.wsite-footer div.paragraph, .wsite-elements.wsite-footer p, .wsite-elements.wsite-footer .product-block .product-title, .wsite-elements.wsite-footer .product-description, .wsite-elements.wsite-footer .wsite-form-field label, .wsite-elements.wsite-footer .wsite-form-field label{}

.wsite-elements.wsite-not-footer:not(.wsite-header-elements) h2, .wsite-elements.wsite-not-footer:not(.wsite-header-elements) .product-long .product-title, .wsite-elements.wsite-not-footer:not(.wsite-header-elements) .product-large .product-title, .wsite-elements.wsite-not-footer:not(.wsite-header-elements) .product-small .product-title, #wsite-content h2, #wsite-content .product-long .product-title, #wsite-content .product-large .product-title, #wsite-content .product-small .product-title, .blog-sidebar h2 {}

#wsite-content h2, #wsite-content .product-long .product-title, #wsite-content .product-large .product-title, #wsite-content .product-small .product-title, .blog-sidebar h2 {}

.wsite-elements.wsite-footer h2, .wsite-elements.wsite-footer .product-long .product-title, .wsite-elements.wsite-footer .product-large .product-title, .wsite-elements.wsite-footer .product-small .product-title{}

#wsite-title {}

.wsite-menu-default a {}

.wsite-menu a {}

.wsite-image div, .wsite-caption {}

.galleryCaptionInnerText {}

.fancybox-title {}

.wslide-caption-text {}

.wsite-phone {}

.wsite-headline,.wsite-header-section .wsite-content-title {}

.wsite-headline-paragraph,.wsite-header-section .paragraph {}

.wsite-button-inner {}

.wsite-not-footer blockquote {}

.wsite-footer blockquote {}

.blog-header h2 a {}

#wsite-content h2.wsite-product-title {}

.wsite-product .wsite-product-price a {}

}</style>



		<script>

var STATIC_BASE = '//cdn1.editmysite.com/';

var ASSETS_BASE = '//cdn2.editmysite.com/';

var STYLE_PREFIX = 'wsite';

</script>

<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.8.3/jquery.min.js"></script>



<script type="text/javascript" src="//cdn2.editmysite.com/js/lang/en/stl.js?buildTime=1551314940&amp;"></script>

<script src="//cdn2.editmysite.com/js/site/main.js?buildTime=1551314940"></script><script type="text/javascript">

		function initCustomerAccountsModels() {

					(function(){_W.setup_rpc({"url":"\/ajax\/api\/JsonRPC\/CustomerAccounts\/","actions":{"CustomerAccounts":[{"name":"login","len":2,"multiple":false,"standalone":false},{"name":"logout","len":0,"multiple":false,"standalone":false},{"name":"getSessionDetails","len":0,"multiple":false,"standalone":false},{"name":"getAccountDetails","len":0,"multiple":false,"standalone":false},{"name":"getOrders","len":0,"multiple":false,"standalone":false},{"name":"register","len":4,"multiple":false,"standalone":false},{"name":"emailExists","len":1,"multiple":false,"standalone":false},{"name":"passwordReset","len":1,"multiple":false,"standalone":false},{"name":"passwordUpdate","len":3,"multiple":false,"standalone":false},{"name":"validateSession","len":1,"multiple":false,"standalone":false}]},"namespace":"_W.CustomerAccounts.RPC"});

_W.setup_model_rpc({"rpc_namespace":"_W.CustomerAccounts.RPC","model_namespace":"_W.CustomerAccounts.BackboneModelData","collection_namespace":"_W.CustomerAccounts.BackboneCollectionData","bootstrap_namespace":"_W.CustomerAccounts.BackboneBootstrap","models":{"CustomerAccounts":{"_class":"CustomerAccounts.Model.CustomerAccounts","defaults":null,"validation":null,"types":null,"idAttribute":null,"keydefs":null}},"collections":{"CustomerAccounts":{"_class":"CustomerAccounts.Collection.CustomerAccounts"}},"bootstrap":[]});

})();

		}

		if(document.createEvent && document.addEventListener) {

			var initEvt = document.createEvent('Event');

			initEvt.initEvent('customerAccountsModelsInitialized', true, false);

			document.dispatchEvent(initEvt);

		} else if(document.documentElement.initCustomerAccountsModels === 0){

			document.documentElement.initCustomerAccountsModels++

		}

		</script>

		<script type="text/javascript"> _W = _W || {}; _W.securePrefix='www.milburnplumbingservices.com.au'; </script><script>_W = _W || {};

			_W.customerLocale = "en_AU";

			_W.storeName = null;

			_W.storeCountry = "AU";

			_W.storeCurrency = "AUD";

			_W.storeEuPrivacyPolicyUrl = "";

			com_currentSite = "528477904287345708";

			com_userID = "95028178";</script><script type="text/javascript">_W.configDomain = "www.weebly.com";</script><script>_W.relinquish && _W.relinquish()</script>

<script type="text/javascript" src="//cdn2.editmysite.com/js/lang/en/stl.js?buildTime=1551314940&amp;"></script><script> _W.themePlugins = [];</script><script type="text/javascript"> _W.recaptchaUrl = "https://www.google.com/recaptcha/api.js"; </script><script type="text/javascript"> window._W = window._W || {}; _W.showV2Footer = 0; </script><script type="text/javascript">

</script>

		<link rel="canonical" href="https://www.youroriginalcontent.com" />



		<script type="text/javascript">var _swloc = (("https:" == document.location.protocol) ? "https://analytics." : "http://analytics.") + "sitewit.com/v3/650647891/sw.js";var _swscript = document.createElement('script');('async' in document.scripts[0] ? _swscript.async = false : null);_swscript.src = _swloc;document.head.appendChild(_swscript);</script><script src="https://analytics.sitewit.com/v3/650647891/sw.js"></script>

	<style type="text/css">.customer-accounts-app__body[data-v-364d7bb4]{width:100%}.customer-accounts-app__modal-wrapper[data-v-364d7bb4]{position:fixed;top:0;left:0;width:100vw;height:100vh;z-index:17;display:flex;flex-direction:column;justify-content:center;align-items:center;background:rgba(0,0,0,.25)}.customer-accounts-app__modal[data-v-364d7bb4]{width:580px;box-sizing:border-box;max-height:100%;min-height:415px;overflow-y:auto;z-index:18;padding:24px;background:#fff;display:flex}.wsite-theme-dark .customer-accounts-app__modal-wrapper[data-v-364d7bb4]{background:hsla(0,0%,100%,.25)}.wsite-theme-dark .customer-accounts-app__modal[data-v-364d7bb4]{background:#000}@media (max-width:600px){.customer-accounts-app__modal[data-v-364d7bb4]{width:100%;height:100%}}</style><style type="text/css">.loading[data-v-09548d63]{display:flex;width:100%;align-items:center;justify-content:center;margin:15px 0}</style><style type="text/css">.page-header__wrapper[data-v-04176401]{display:flex;flex-direction:row;align-items:center;justify-content:space-between}.page-header__text[data-v-04176401]{margin:0 32px 0 0}.page-header__divider[data-v-04176401]{height:1px;width:100%;margin:24px 0;background:rgba(0,0,0,.1)}</style><style type="text/css">.form-field[data-v-64b6436e]{margin:16px 0}.form-field__label[data-v-64b6436e]{display:flex;flex-direction:column}.form-field__title[data-v-64b6436e]{margin-bottom:8px;font-size:90%;color:#3b526d}.wsite-theme-dark .form-field__title[data-v-64b6436e]{color:#dfe6ee}.form-field__error[data-v-64b6436e]{margin-top:14px}</style><style type="text/css">.field-error[data-v-60fd9ad5]{color:#ff2825}</style><style type="text/css">.form-instructions[data-v-1b20c747]{margin-bottom:24px}</style><style type="text/css">.site-input[data-v-7c9d3469]{border:1px solid #dbdbdb;padding:8px;border-radius:2px}.site-input__is-invalid[data-v-7c9d3469]{border:1px solid #ff2825}</style><style type="text/css">.actions-row[data-v-e3a0cd02]{margin-top:24px;display:flex;flex-direction:row;justify-content:space-between;align-items:center}.actions-row div[data-v-e3a0cd02]:first-child{margin-right:16px}</style><style type="text/css">.back-button[data-v-089f4bca]{display:flex;flex-direction:row;align-items:center}.back-button__icon[data-v-089f4bca]{margin:0 8px 0 0}</style><style type="text/css">p[data-v-5e8be7b5]{font-weight:700}.order-confirmation__checkbox[data-v-5e8be7b5]{margin-right:6px}.order-confirmation__success-message[data-v-5e8be7b5]{display:flex;flex-direction:row;align-items:center}</style><style type="text/css">.register__privacy-policy[data-v-00b778c0]{margin:24px 0;font-size:small}</style><style type="text/css">.customer-accounts-app__body[data-v-a6d2e276]{width:100%}.customer-accounts-app__modal-wrapper[data-v-a6d2e276]{position:fixed;top:0;left:0;width:100vw;height:100vh;z-index:17;display:flex;flex-direction:column;justify-content:center;align-items:center;background:rgba(0,0,0,.25)}.customer-accounts-app__modal[data-v-a6d2e276]{width:580px;box-sizing:border-box;max-height:100%;min-height:415px;overflow-y:auto;z-index:18;padding:24px;background:#fff;display:flex}.wsite-theme-dark .customer-accounts-app__modal-wrapper[data-v-a6d2e276]{background:hsla(0,0%,100%,.25)}.wsite-theme-dark .customer-accounts-app__modal[data-v-a6d2e276]{background:#000}@media (max-width:600px){.customer-accounts-app__modal[data-v-a6d2e276]{width:100%;height:100%}}</style><style type="text/css">.account-details__wrapper[data-v-4c887ac0]{width:100%}.account-details__group[data-v-4c887ac0]{margin-bottom:24px;text-align:left}.account-details__table-container[data-v-4c887ac0]{overflow:auto}.account-details__title[data-v-4c887ac0]{text-align:left;margin-bottom:6px}.account-details__log-out[data-v-4c887ac0]{cursor:pointer}.account-details__error[data-v-4c887ac0]{margin:36px 0;text-align:left}.error[data-v-4c887ac0]{color:#ff2825}</style><style type="text/css">.section-header__wrapper[data-v-1e785cc9]{display:flex;flex-direction:row;align-items:center;justify-content:space-between}.section-header__text[data-v-1e785cc9]{margin:24px 0}</style><style type="text/css">.empty-content-area__container[data-v-8fbca36e]{display:flex;flex-direction:row;align-items:center;justify-content:center;margin-bottom:32px;width:100%;min-height:80px;background:#f2f2f2}.wsite-theme-dark .empty-content-area__container[data-v-8fbca36e]{background:#000}</style><style type="text/css">.order-details__table[data-v-244ed988]{width:100%;margin-top:12px;margin-bottom:36px;empty-cells:show;border-collapse:collapse;border-spacing:0;border:none}.order-details__table th[data-v-244ed988]{margin:0;padding:16px;text-align:left;background:#f2f2f2}.wsite-theme-dark .order-details__table th[data-v-244ed988]{background:#000}.order-details__table td[data-v-244ed988]{margin:0;padding:16px;text-align:left}</style><style type="text/css">.shipping-address__wrapper[data-v-4fc55e12]{text-align:left;width:100%}</style><style type="text/css">.reset-password__link[data-v-72d92d0c]{cursor:pointer}</style><script src="https://connect.sitewit.com/js/650647891/sw_connect.js?&amp;ns=sw"></script></head>

	<body class="header-page  wsite-theme-dark  wsite-page-index" style="padding-bottom: 2.8em;"><div class="wrapper">

    <div class="cento-header">

      <div class="container">

        <div class="logo"><span class="wsite-logo">



	<a href="/">

	

	<span id="wsite-title">Milburn Plumbing Services</span>

	

	</a>



</span></div>

        <label class="hamburger"><span></span></label>

      </div>

      <div class="nav-wrap">

        <div class="container">

          <div class="nav desktop-nav"><ul class="wsite-menu-default">

		<li id="active" class="wsite-menu-item-wrap wsite-nav-1" style="position: relative;">

			<a href="/" class="wsite-menu-item" style="position: relative;">

				Home

			</a>

			

		</li>

		<li id="pg572356952646405851" class="wsite-menu-item-wrap wsite-nav-2" style="position: relative;">

			<a href="/contact.html" class="wsite-menu-item" style="position: relative;">

				Contact

			</a>

			

		</li>

		<li id="pg955264036985678484" class="wsite-menu-item-wrap wsite-nav-3" style="position: relative;">

			<a href="/feedback.html" class="wsite-menu-item" style="position: relative;">

				Feedback

			</a>

			

		</li>

</ul>

</div>

        </div>

      </div>

    </div>



    <div class="banner-wrap">

      <div class="wsite-elements wsite-not-footer wsite-header-elements">

	<div class="wsite-section-wrap">

	<div class="wsite-section wsite-header-section wsite-section-bg-color" style="height: auto;background-color: #044E76;background-image: none;is_customized: 1;">

		<div class="wsite-section-content">

			

        <div class="container">

          <div class="banner">

				<div class="wsite-section-elements">

					<h2 class="wsite-content-title">For Your Plumbing And Gasfitting Needs</h2>



<div class="paragraph"><a href="tel:0455662000"> Call Now 0455 662 000</a></div>

				</div>

			</div>

        </div>

      

		</div>

		<div class=""></div>

	</div>

</div>



</div>



    </div>



    <div class="content-wrap">

      <div id="wsite-content" class="wsite-elements wsite-not-footer">

	<div class="wsite-section-wrap">

	<div class="wsite-section wsite-body-section wsite-section-bg-image wsite-background-18" style="background-image: url(&quot;/uploads/9/5/0/2/95028178/background-images/1327542669.jpg&quot;) ;background-repeat: no-repeat ;background-position: 50% 50% ;background-size: 100% ;background-color: transparent ;background-size: cover;">

		<div class="wsite-section-content">

        <div class="container"><h1>Milburn Plumbing Services</h1>

			<div class="wsite-section-elements">

				<div id="565431570965023284"><div><style type="text/css">

	#element-cff44f0b-d7c0-4677-bce6-3891eb29d99e a.back-to-top {

  display: none;

  width: 40px;

  height: 40px;

  text-indent: -9999px;

  position: fixed;

  z-index: 999;

  bottom: 40px;

  background: #0672AC url("//marketplace.editmysite.com/elements/569198776394566218-1.0.1/assets/up-arrow.png") no-repeat center 43%;

  -webkit-border-radius: 20px;

  -moz-border-radius: 20px;

  border-radius: 20px;

}

#element-cff44f0b-d7c0-4677-bce6-3891eb29d99e a.back-to-top.back-to-top-left {

  left: 40px;

}

#element-cff44f0b-d7c0-4677-bce6-3891eb29d99e a.back-to-top.back-to-top-right {

  right: 40px;

}



</style><div id="element-cff44f0b-d7c0-4677-bce6-3891eb29d99e" data-platform-element-id="569198776394566218-1.0.1" class="platform-element-contents">

	<a href="#" class="back-to-top back-to-top-right"></a>



</div>

<div style="clear:both;"></div><script type="text/javascript" class="element-script">function setupElement565431570965023284() {

	var requireFunc = window.platformElementRequire || window.require;



	// Relies on a global require, specific to platform elements

	requireFunc([

		'w-global',

		'underscore',

		'jquery',

		'backbone',

		'util/platform/elements/PlatformElement',

		'util/platform/elements/PlatformElementSettings'

	], function(

		_W,

		_,

		$,

		Backbone,

		PlatformElement,

		PlatformElementSettings

	) {

		var dependencies = null || [];

		var platform_element_id = "569198776394566218-1.0.1";



		if (typeof _W.loadedPlatformDependencies === 'undefined') {

			_W.loadedPlatformDependencies = [];

		}



		if (typeof _W.platformElements === 'undefined') {

			_W.platformElements = [];

		}



		if (typeof _W.platformElements[platform_element_id] === 'undefined') {

			_W.platformElements[platform_element_id] = {};

			_W.platformElements[platform_element_id].deferredObject = new $.Deferred();

			_W.platformElements[platform_element_id].deferredPromise = _W.platformElements[platform_element_id].deferredObject.promise();

		}



		if(_.intersection(_W.loadedPlatformDependencies, dependencies).length !== dependencies.length){

			_.reduce(dependencies, function(promise, nextScript){

				_W.loadedPlatformDependencies.push(nextScript);

				return promise.then(function(){

					return $.getScript(nextScript);

				});

			}, $().promise()).then(function(){

				_W.platformElements[platform_element_id].deferredObject.resolve();

			});

		}



		if (dependencies.length === 0){

			_W.platformElements[platform_element_id].deferredObject.resolve();

		}



		_W.platformElements[platform_element_id].deferredPromise.done(function(){

			var _ElementDefinition = PlatformElement.extend({

    initialize: function(options) {

	    this.amountScrolled = 300;



	    var self = this;



	    $(window).bind('scroll', function (ev) {

		    self.checkAppear();

	    });

    },



    events: {

    	"click a.back-to-top": "goBackToTop"

    },



	goBackToTop: function() {



		var animateTimeMS = 1000 - (parseInt(this.settings.get("scroll_speed")) * 100);



		$('html, body').animate({

			scrollTop: 0

		}, animateTimeMS);

		

		return false;

	},



	checkAppear: function() {

		if ( $(window).scrollTop() > this.amountScrolled ) {

			$('a.back-to-top').fadeIn('slow');

		} else {

			$('a.back-to-top').fadeOut('slow');

		}

	}

});



;



			if (typeof _ElementDefinition == 'undefined' || typeof _ElementDefinition == 'null') {

				var _ElementDefinition = PlatformElement.extend({});

			}



			var _Element = _ElementDefinition.extend({

				initialize: function() {

					// we still want to call the initialize function defined by the developer

					// however, we don't want to call it until placeholders have been replaced

					this.placeholderInterval = setInterval(function() {

						// so use setInterval to check for placeholders.

						if (this.$('.platform-element-child-placeholder').length == 0) {

							clearInterval(this.placeholderInterval);

							this.constructor.__super__.initialize.apply(this);

						}

					}.bind(this), 100);

				}

			});



			_Element.prototype.settings = new PlatformElementSettings({"icon_size_each":[{"icon_size_index":0},{"icon_size_index":1},{"icon_size_index":2},{"icon_size_index":3},{"icon_size_index":4},{"icon_size_index":5},{"icon_size_index":6},{"icon_size_index":7},{"icon_size_index":8},{"icon_size_index":9},{"icon_size_index":10},{"icon_size_index":11},{"icon_size_index":12},{"icon_size_index":13},{"icon_size_index":14},{"icon_size_index":15},{"icon_size_index":16},{"icon_size_index":17},{"icon_size_index":18},{"icon_size_index":19},{"icon_size_index":20},{"icon_size_index":21},{"icon_size_index":22},{"icon_size_index":23},{"icon_size_index":24},{"icon_size_index":25},{"icon_size_index":26},{"icon_size_index":27},{"icon_size_index":28},{"icon_size_index":29},{"icon_size_index":30},{"icon_size_index":31},{"icon_size_index":32},{"icon_size_index":33},{"icon_size_index":34},{"icon_size_index":35},{"icon_size_index":36},{"icon_size_index":37},{"icon_size_index":38},{"icon_size_index":39}],"scroll_speed_each":[{"scroll_speed_index":0},{"scroll_speed_index":1},{"scroll_speed_index":2}],"margin_bottom_each":[{"margin_bottom_index":0},{"margin_bottom_index":1},{"margin_bottom_index":2},{"margin_bottom_index":3},{"margin_bottom_index":4},{"margin_bottom_index":5},{"margin_bottom_index":6},{"margin_bottom_index":7},{"margin_bottom_index":8},{"margin_bottom_index":9},{"margin_bottom_index":10},{"margin_bottom_index":11},{"margin_bottom_index":12},{"margin_bottom_index":13},{"margin_bottom_index":14},{"margin_bottom_index":15},{"margin_bottom_index":16},{"margin_bottom_index":17},{"margin_bottom_index":18},{"margin_bottom_index":19},{"margin_bottom_index":20},{"margin_bottom_index":21},{"margin_bottom_index":22},{"margin_bottom_index":23},{"margin_bottom_index":24},{"margin_bottom_index":25},{"margin_bottom_index":26},{"margin_bottom_index":27},{"margin_bottom_index":28},{"margin_bottom_index":29},{"margin_bottom_index":30},{"margin_bottom_index":31},{"margin_bottom_index":32},{"margin_bottom_index":33},{"margin_bottom_index":34},{"margin_bottom_index":35},{"margin_bottom_index":36},{"margin_bottom_index":37},{"margin_bottom_index":38},{"margin_bottom_index":39}],"margin_horizontal_each":[{"margin_horizontal_index":0},{"margin_horizontal_index":1},{"margin_horizontal_index":2},{"margin_horizontal_index":3},{"margin_horizontal_index":4},{"margin_horizontal_index":5},{"margin_horizontal_index":6},{"margin_horizontal_index":7},{"margin_horizontal_index":8},{"margin_horizontal_index":9},{"margin_horizontal_index":10},{"margin_horizontal_index":11},{"margin_horizontal_index":12},{"margin_horizontal_index":13},{"margin_horizontal_index":14},{"margin_horizontal_index":15},{"margin_horizontal_index":16},{"margin_horizontal_index":17},{"margin_horizontal_index":18},{"margin_horizontal_index":19},{"margin_horizontal_index":20},{"margin_horizontal_index":21},{"margin_horizontal_index":22},{"margin_horizontal_index":23},{"margin_horizontal_index":24},{"margin_horizontal_index":25},{"margin_horizontal_index":26},{"margin_horizontal_index":27},{"margin_horizontal_index":28},{"margin_horizontal_index":29},{"margin_horizontal_index":30},{"margin_horizontal_index":31},{"margin_horizontal_index":32},{"margin_horizontal_index":33},{"margin_horizontal_index":34},{"margin_horizontal_index":35},{"margin_horizontal_index":36},{"margin_horizontal_index":37},{"margin_horizontal_index":38},{"margin_horizontal_index":39}],"background_color":"#0672AC","icon_size":40,"scroll_speed":3,"margin_bottom":40,"left_or_right":"right","margin_horizontal":40});

			_Element.prototype.settings.page_element_id = "565431570965023284";



			_Element.prototype.element_id = "cff44f0b-d7c0-4677-bce6-3891eb29d99e";

			_Element.prototype.user_id = "95028178";

			_Element.prototype.site_id = "528477904287345708";

			_Element.prototype.assets_path = "//marketplace.editmysite.com/elements/569198776394566218-1.0.1/assets/";

			new _Element({

				el: '#element-cff44f0b-d7c0-4677-bce6-3891eb29d99e'

			});

		});

	});



}



if (typeof document.documentElement.appReady == 'undefined') {

	document.documentElement.appReady = 0;

}



if (document.documentElement.appReady || (window.inEditor && window.inEditor())) {

	setupElement565431570965023284();

} else if (document.createEvent && document.addEventListener) {

	document.addEventListener('appReady', setupElement565431570965023284, false);

} else {

	document.documentElement.attachEvent('onpropertychange', function(event){

		if (event.propertyName == 'appReady') {

			setupElement565431570965023284();

		}

	});

}



</script></div></div>



<h2 class="wsite-content-title" style="text-align:center;">Our Prompt Services</h2>



<div class="paragraph">Our services can - alter, construct, install, maintain, repair, replace, test or commission - most appliances, fixtures and associated piping and flues of systems that require Plumbing or Gas-fitting. We also unblock Sewer and Stormwater drainage systems.<br /> <br />Meet Gary Milburn, the owner and name sake of Milburn Plumbing Services. He is a licensed and qualified plumber and gas fitter who has 39 years experience in seven aspects of plumbing and gas fitting, to bring to your assistance. Gary prides himself on providing an honest friendly Quality service and effective solutions for his customer&#39;s concerns. Gary will also answer any questions or needs you may have about the task and talk you through the process of the repair or installation needed to resolve your plumbing or gas fitting problem. Gary also places great importance on customer satisfaction, offering to clean up any mess created during the course of the repair or installation as a standard part of the service. Leaving the area as he found it or better.<br /><br />Our Plumbing and Gas-fitting Services plus advice, is based best in Morwell and will be serving the surrounding rural and township areas, within a 40 kilometre radius of Morwell.</div>



<div class="wsite-spacer" style="height:50px;"></div>



<div><div class="wsite-multicol"><div class="wsite-multicol-table-wrap" style="margin:0 -15px;">

	<table class="wsite-multicol-table">

		<tbody class="wsite-multicol-tbody">

			<tr class="wsite-multicol-tr">

				<td class="wsite-multicol-col" style="width:25%; padding:0 15px;">

					

						



<div><div class="wsite-image wsite-image-border-none " style="padding-top:10px;padding-bottom:10px;margin-left:0px;margin-right:0px;text-align:center">

<a>

<img src="/uploads/9/5/0/2/95028178/plumbing-licence-min_4_orig.jpg" alt="Gary Milburn, licence number 38742" style="width:auto;max-width:100%" />

</a>

<div style="display:block;font-size:90%"></div>

</div></div>





					

				</td>				<td class="wsite-multicol-col" style="width:25%; padding:0 15px;">

					

						



<h2 class="wsite-content-title" style="text-align:center;"><font size="5">Main Classes</font></h2>



<div class="paragraph"><ul><li> Drainage</li><li>Gas fitting</li><li>Irrigation (non-agricultural)</li><li>Mechanical services</li><li>Roofing (storm water)</li><li>Sanitary</li><li> Water supply</li></ul></div>





					

				</td>				<td class="wsite-multicol-col" style="width:25%; padding:0 15px;">

					

						



<h2 class="wsite-content-title" style="text-align:center;"><font size="5">Restricted Classes</font></h2>



<div class="paragraph"><ul><li>Hydrants &amp; hose reels</li><li>Gas disconnect/reconnect</li><li>Gas restricted to type A appliances</li><li>Gas mobile homes</li><li>Solid fuel heaters</li><li>Duct fixing</li><li>Single head split systems</li><li>Class 10A roofing</li><li>Domestic hot water services</li><li>Refrig air cond (basic)</li></ul></div>





					

				</td>				<td class="wsite-multicol-col" style="width:25%; padding:0 15px;">

					

						



<h2 class="wsite-content-title" style="text-align:center;"><font size="5">Specialised Classes</font></h2>



<div class="paragraph"><ul><li>Gas servicing Type A</li></ul></div>





					

				</td>			</tr>

		</tbody>

	</table>

</div></div></div>

			</div>

		</div>

      </div>



	</div>

</div>



</div>



    </div>



    <div class="footer-wrap">

      <div class="container">

     		<div><div class="wsite-elements wsite-footer">

<h2 class="wsite-content-title">Our Plumbing Services Attend To Your Needs Within a 40km Radius of Morwell</h2>



<h2 class="wsite-content-title"> <a href="tel:0455662000"> Call Now 0455 662 000</a></h2>



<div class="paragraph" style="text-align:center;"><font color="#666666" size="3">Copyright Milburn Plumbing Services © 2019 ABN : 26 922 629 443 </font></div></div></div>

      </div>

    </div>

	</div>



  <div class="nav mobile-nav"><ul class="wsite-menu-default">

		<li id="active" class="wsite-menu-item-wrap">

			<a href="/" class="wsite-menu-item">

				Home

			</a>

			

		</li>

		<li id="pg572356952646405851" class="wsite-menu-item-wrap">

			<a href="/contact.html" class="wsite-menu-item">

				Contact

			</a>

			

		</li>

		<li id="pg955264036985678484" class="wsite-menu-item-wrap">

			<a href="/feedback.html" class="wsite-menu-item">

				Feedback

			</a>

			

		</li>

</ul>

</div>



  <script>(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o), m=s.getElementsByTagName(o)[0]; 

  a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','https://www.google-analytics.com/analytics.js','ga'); ga('create', 'UA-88378966-1', 'auto'); ga('send', 'pageview');</script>

  <script type="text/javascript" src="/files/theme/plugins.js?1539763099"></script>

  <script type="text/javascript" src="/files/theme/custom.js?1539763099"></script>

    <div id="customer-accounts-app"></div>

    <script src="//cdn2.editmysite.com/js/site/main-customer-accounts-site.js?buildTime=1551314940"></script>



		

<script async="" src="https://www.googletagmanager.com/gtag/js?id=UA-88378966-1"></script>

<script>

window.dataLayer = window.dataLayer || [];

function gtag(){dataLayer.push(arguments);}

gtag('js', new Date());



gtag('config', 'UA-88378966-1', { 'optimize_id': 'GTM-WX8BKJW'});

</script>



<script type="text/javascript">

	var _gaq = _gaq || [];

	_gaq.push(['_setAccount', 'UA-7870337-1']);

	_gaq.push(['_setDomainName', 'none']);

	_gaq.push(['_setAllowLinker', true]);



	(function() {

		var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;

		ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';

		var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);

	})();



	_W.Analytics = _W.Analytics || {'trackers': {}};

	_W.Analytics.trackers.wGA = '_gaq';

</script>



<script type="text/javascript" async="1">

	;(function(p,l,o,w,i,n,g){if(!p[i]){p.GlobalSnowplowNamespace=p.GlobalSnowplowNamespace||[];

			p.GlobalSnowplowNamespace.push(i);p[i]=function(){(p[i].q=p[i].q||[]).push(arguments)

			};p[i].q=p[i].q||[];n=l.createElement(o);g=l.getElementsByTagName(o)[0];n.async=1;

			n.src=w;g.parentNode.insertBefore(n,g)}}(window,document,'script','//cdn2.editmysite.com/js/wsnbn/snowday262.js','snowday'));



	var r = [99, 104, 101, 99, 107, 111, 117, 116, 46, 40, 119, 101, 101, 98, 108, 121, 124, 101, 100, 105, 116, 109, 121, 115, 105, 116, 101, 41, 46, 99, 111, 109];

	var snPlObR = function(arr) {

		var s = '';

		for (var i = 0 ; i < arr.length ; i++){

			s = s + String.fromCharCode(arr[i]);

		}

		return s;

	};

	var s = snPlObR(r);



	var regEx = new RegExp(s);



	_W.Analytics = _W.Analytics || {'trackers': {}};

	_W.Analytics.trackers.wSP = 'snowday';

	_W.Analytics.user_id = '95028178';

	_W.Analytics.site_id = '528477904287345708';



	// Setting do not track if the GDPR cookie is not present. This is then checked by the snowday initializer

	// to set tracking decisions. https://github.com/snowplow/snowplow-javascript-tracker/blob/2.6.2/src/js/tracker.js#L1509

	window.doNotTrack = document.cookie.indexOf('gdpr-kb') === -1 ? 'yes' : null;





	(function(app_id, ec_hostname, discover_root_domain) {

		var track = window[_W.Analytics.trackers.wSP];

		if (!track) return;

		track('newTracker', app_id, ec_hostname, {

			appId: app_id,

			post: true,

			platform: 'web',

			discoverRootDomain: discover_root_domain,

			cookieName: '_snow_',

			contexts: {

				webPage: true,

				performanceTiming: true,

				gaCookies: true

			},

			crossDomainLinker: function (linkElement) {

				return regEx.test(linkElement.href);

			},

			respectDoNotTrack: document.cookie.indexOf('gdpr-kb') === -1

		});

		track('trackPageView', _W.Analytics.user_id+':'+_W.Analytics.site_id);

		track('crossDomainLinker', function (linkElement) {

			return regEx.test(linkElement.href);

		});

	})(

		'_wn',

		'ec.editmysite.com',

		true

	);

</script>







<script>

	(function(jQuery) {

		try {

			if (jQuery) {

				jQuery('div.blog-social div.fb-like').attr('class', 'blog-social-item blog-fb-like');

				var $commentFrame = jQuery('#commentArea iframe');

				if ($commentFrame.length > 0) {

					var frameHeight = jQuery($commentFrame[0].contentWindow.document).height() + 50;

					$commentFrame.css('min-height', frameHeight + 'px');

				}

				if (jQuery('.product-button').length > 0){

					jQuery(document).ready(function(){

						jQuery('.product-button').parent().each(function(index, product){

							if(jQuery(product).attr('target') == 'paypal'){

								if (!jQuery(product).find('> [name="bn"]').length){

									jQuery('<input>').attr({

										type: 'hidden',

										name: 'bn',

										value: 'DragAndDropBuil_SP_EC'

									}).appendTo(product);

								}

							}

						});

					});

				}

			}

			else {

				// Prototype

				$$('div.blog-social div.fb-like').each(function(div) {

					div.className = 'blog-social-item blog-fb-like';

				});

				$$('#commentArea iframe').each(function(iframe) {

					iframe.style.minHeight = '410px';

				});

			}

		}

		catch(ex) {}

	})(window._W && _W.jQuery);



	window._W.isEUUser = false;

	window._W.showCookieToAll = "all";

</script>





	



<div><div class="kb-container" id="kb-container">

    <div class="kb-content">

        <div class="kb-description">This site uses cookies to personalize your experience, analyze site usage, and offer tailored promotions. <a style="color: #2990ea; text-decoration: none;" href="https://www.youronlinechoices.eu" target="_blank">www.youronlinechoices.eu</a></div>

        <a href="#" class="kb-remind-link" id="kb-link">Remind me later</a>

        <a href="#" class="kb-notice-btn" id="kb-btn">I accept</a>

    </div>

</div>



<style>

    .kb-container {

        background: #363b3e;

        font-size: 0.8em;

        text-align: center;

        line-height: 1.3;

        position: fixed;

        z-index: 9999999999; /* make sure z-index is higher than all chat apps  */

        bottom: 0;

        width: 100vw;

        border-top: #fff 3px solid;

        font-family: "Graphik","effra","Proxima Nova","Helvetica Neue","Helvetica","Arial",sans-serif;

    }

    .kb-content {

        color: #fff;

        padding: 10px 0px;

        width: 100%;

        max-width: 1440px;

        margin: auto;

    }



    @media screen and (min-width: 64.0625em) {

        .kb-content {

            padding: 5px 20px;

        }

    }

    .kb-description {

        text-align: left;

        line-height: 1.3;

        width: 70%;

        display: inline-block;

        vertical-align: middle;

    }

    @media screen and (min-width: 40.0625em) {

        .kb-description {

            margin-right: 2em;

        }

        .kb-remind-link {

            margin-left: -2em;

        }

    }



    a.kb-notice-btn {

        color: #fff;

        background: #0073ff;

        display: inline-block;

        padding: 10px 0;

        border-radius: 3px;

        font-weight: 500;

        min-width: 150px;

        text-decoration: none;

    }



    a.kb-notice-btn:hover {

        background-color: #fff;

        color: #0073ff;

    }



    .kb-remind-link {

        color: #2990ea;

        text-decoration: none;

        margin-right: 1em;

    }



    @media screen and (max-width: 40em) {

        .kb-notice-btn {

            width: 25%;

            min-width: unset;

        }

    }

}

</style>

</div><div id="wsite-menus"></div></body></html>

**Comment by GaryMilburn** [comment]
fix issues
